src/include/btmem.h

```diff
@@ -744,12 +744,13 @@ struct __wt_ref {
 #define	ref_ikey	key.ikey
 
 	WT_PAGE_DELETED	*page_del;	/* Deleted on-disk page information */
+	bool has_las_records;
```

Comment:
@keithbostic I'm hoping (hard) that there is another place to store this state in the WT_REF. We just need one bit. I'd really rather not introduce a new state, but that's probably preferable to extending the size by 8 bytes.